great3sims/psf.py

```diff
@@ -1382,10 +1382,22 @@ class DrawPSFBuilder(PSFBuilder):
     def __init__(self, obs_type, multiepoch, shear_type, draw_psf_src):
         self.obs_type = obs_type
         self.draw_psf_src = os.path.abspath(draw_psf_src)
+        filename = os.path.basename(self.draw_psf_src)
+        fin = open(self.draw_psf_src, "r")
+        #   The input file draw_psf_src is a file which contains a list of all of the available Psfs
+        #   It contains the name of the mini-library which contains the psf, the "GroupIndex" which
+        #   is the numeric index of the library, the "PsfIndex", which is the hdu index in the mini-
+        #   library, and the "PsfNumber", which is the index of the star in the PhoSim pars file
+        self.psfInfos = []
+        while True:
```

Comment:
Again, it would be much nicer to use a more idiomatic loop, perhaps with a context manager for the open file:
```python
with open(self.draw_psf_src, 'r") as fin:
    for line in fin:
        entry = line.split()
        # etc
```